### ISSUE TITLE
[Bug] Gravity Core not remebering gravity field size after a reboot.

### ISSUE THREAD

**Issue by Sullos** [tsunamayo/Starship-EVO#1814]
### Short Description of the issue:

The gravity core doesn't reproduce the values set in the core after a game reboot.

### Steps to reproduce the issue:
Use a ship somewhere outside of the effects of the torus gravity field. 
Make sure the ship is bigger than the default 8x8 gravity area. 
Make the gravity field bigger than the default value.
Upon rebooting the game, you should find that if you don't have antigrav mode on, that if you go further than 8 blocks from the core, gravity stops working right.
Using a switch to deactivate and reactivate the gravity core will fix the problem, the same as reentering your custom values into the gravity core.

Basically, it seems that the custom values in the gravity core aren't activated by default upon restarting the game. You have to either reenter them or reactivate the gravity core for your custom dimensions to take effect.

### Environment
- Skywanderers version: 20w17b
- Graphics Card: 980ti

**Comment by tsunamayo** [comment]
Thanks for spotting that one, issue will be fixed for 20w17d. Thanks!
